Code/Editor/Util/PathUtil.cpp

```diff
@@ -255,16 +245,16 @@ namespace Path
     /// Get the data folder
     AZStd::string GetEditingGameDataFolder()
     {
+        static AZStd::string s_currentModName;
```

Comment:
should be, I didnt get any issues/crashes with it

Reply:
Actually it would be safer to use an AZStd::fixed_string here.

The here allocated to the static AZStd::string will still be in scope after the SystemAllocator has tore down.

Reply:
replacing with a fixed_string

Reply:
takins a second look to the method `s_currentModName` is always empty, I think the intention here was to be able to pass a mod path, will replace with a fixed_string
I am also going to go through all the cases of `static AZStd::string`